test/e2e/ctx/provisioner_kubeconfig.go

```diff
@@ -1,3 +1,4 @@
+//go:build !kind
```

Comment:
unrelated: This points to the need for a shared operator-framework/test-infra repository so we're not de-duplicating work across the registry and OLM repositories.